installer/conf/td-agent-bit.conf

```diff
@@ -0,0 +1,53 @@
+[SERVICE]
+    Flush         1

```

Comment:
why is this 1 sec ? we want to flush every sec ? looks default is 5 (which we should keep unless there is a reason to change)